utility/src/test/java/org/oppia/android/util/utility/NetworkConnectionTestUtil.kt

```diff
@@ -0,0 +1,35 @@
+package org.oppia.android.util.utility
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
+import org.robolectric.Shadows
+import org.robolectric.shadows.ShadowNetworkInfo
+
+/** Test utility to modify [ShadowNetworkInfo] during the test. */
+class NetworkConnectionTestUtil {
```

Comment:
I suggest making this injectable & have it inject its own context rather than using a companion object & passing it in.

Reply:
Done.